orbit/pkg/constant/constant_notwindows.go

```diff
@@ -1,4 +1,5 @@
 //+build !windows
+//go:build !windows
```

Comment:
This is solved in `main` now, fwiw.

Reply:
oh cool, I'll close this and rebase my other PR, thanks!